akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,70 @@
+jQuery(document).ready(function ($) {
+
+  var latestInSeries = {
+    "2.2": {
+      latest: "2.2.5",
+      deprecated: true,
+      instead: "2.3"
+    },
+    "2.3": {
+      latest: "2.3.13"
+    },
+    "2.4": {
+      latest: "2.4.0-RC1"
+    },
+
+    // -- RP --
+    "rp-15v01": {
+      rp: true,
+      latest: "rp-15v01p05"
+    }
+  };
+
+  //if (location.hostname === "doc.akka.io") {
+  for (var series in latestInSeries) {
+    if (browsedVersion().startsWith(series)) {
+      floatyWarning(browsedVersion(), latestInSeries[series])
+    }
+  }
+  //}
+
+  function floatyWarning(version, seriesInfo) {
+    var targetUrl = ("" + window.location).replace(version, seriesInfo.latest);
+    var $floatyWarning = $('<div id="floaty-warning"/>');
+
+    $floatyWarning
+        .html('' +
+        'You are browsing the docs for Akka [' + version + '], ' +
+        'however the latest release in this series is: ' +
+        '<strong><a href="' + targetUrl + '">[' + seriesInfo.latest + ']</a></strong>. <br/>' +
+        '<a href="' + targetUrl + '">' +
+        '<string>Click here to go to the latest [' + seriesInfo.latest + '] version of the docs.</strong>' +
+        '</a>');
+
+    if (!!seriesInfo['deprecated']) { // yes !!, because: undefined => false, false => false, true => true
```

Comment:
Yeah that is true I guess. I'm not a bit fan of using falsy values, always convert explicitly to boolean (been coding surprisingly much js at some point...).